tests/php/test-class-amp-theme-support.php

```diff
@@ -2261,6 +2261,7 @@ public function test_prepare_response_doing_template_actions() {
 		};
 		foreach ( [ 'wp_head', 'wp_footer', 'amp_post_template_head', 'amp_post_template_footer' ] as $action ) {
 			$wp_actions = [];
+			wp();
```

Comment:
@westonruter Please note the setting up WordPress query here. We are using `add_mobile_alternative_link` on `wp_head` and it uses `amp_is_request`. So resetting `$wp_actions` causes this error due to the unavailability of the WP query  :

![image](https://user-images.githubusercontent.com/54371619/203466791-0319b6b1-4579-45e2-b4c5-5827ce3bbc7d.png)

Can you please confirm if using `wp()` here is right move?

Reply:
I don't recall all why I did what I did in fad295b4aef5c03cc1805a57673d16f41ae40f3d, but since this method calls `wp()` at the beginning, then adding it here would seem to make sense.